service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json

```diff
@@ -0,0 +1,761 @@
+    "vmImageVersion": "latest",
+    "vnetID": "[resourceId('Microsoft.Network/virtualNetworks',variables('virtualNetworkName'))]",
+    "subnet1Ref": "[concat(variables('vnetID'),'/subnets/',variables('subnet1Name'))]",
+    "supportLogStorageAccountName": "[concat(parameters('clusterName'),'2')]",
+    "applicationDiagnosticsStorageAccountName": "[concat(parameters('clusterName'),'3')]",
+    "lbID0": "[resourceId('Microsoft.Network/loadBalancers',concat(variables('lbName'),'-','0'))]",
+    "lbIPConfig0": "[concat(variables('lbID0'),'/frontendIPConfigurations/LoadBalancerIPConfig')]",
+    "lbPoolID0": "[concat(variables('lbID0'),'/backendAddressPools/LoadBalancerBEAddressPool')]",
+    "lbProbeID0": "[concat(variables('lbID0'),'/probes/FabricGatewayProbe')]",
+    "lbHttpProbeID0": "[concat(variables('lbID0'),'/probes/FabricHttpGatewayProbe')]",
+    "vmNodeType0Name": "[concat('NT1', variables('vmName'))]",
+    "vmNodeType0Size": "Standard_A2"
+  },
+  "resources": [
+    {
+      "apiVersion": "2015-05-01-preview",
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[variables('vmStorageAccountName')]",
+      "location": "[variables('computeLocation')]",
+      "properties": {
+        "accountType": "[parameters('storageAccountType')]"
+      },
+      "tags": {
+        "resourceType": "Service Fabric",
+        "clusterName": "[parameters('clusterName')]"
+      }
+    },
+    {
+      "apiVersion": "2015-05-01-preview",
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[variables('supportLogStorageAccountName')]",
+      "location": "[variables('computeLocation')]",
+      "properties": {
+        "accountType": "[parameters('supportLogStorageAccountType')]"
+      },
+      "tags": {
+        "resourceType": "Service Fabric",
+        "clusterName": "[parameters('clusterName')]"
+      }
+    },
+    {
+      "apiVersion": "2015-05-01-preview",
+      "type": "Microsoft.Storage/storageAccounts",
+      "name": "[variables('applicationDiagnosticsStorageAccountName')]",
+      "location": "[variables('computeLocation')]",
+      "properties": {
+        "accountType": "[parameters('applicationDiagnosticsStorageAccountType')]"
+      },
+      "tags": {
+        "resourceType": "Service Fabric",
+        "clusterName": "[parameters('clusterName')]"
+      }
+    },
+    {
+      "apiVersion": "2015-05-01-preview",
+      "type": "Microsoft.Network/virtualNetworks",
+      "name": "[variables('virtualNetworkName')]",
+      "location": "[variables('computeLocation')]",
+      "properties": {
+        "addressSpace": {
+          "addressPrefixes": [
+            "[variables('addressPrefix')]"
+          ]
+        },
+        "subnets": [
+          {
+            "name": "[variables('subnet1Name')]",
+            "properties": {
+              "addressPrefix": "[variables('subnet1Prefix')]"
+            }
+          },
+          {
+            "name": "[variables('subnet2Name')]",
```

Comment:
Wont Fix - This is so that customers can add any other resources they want to that subnet.. We want to avoid folks adding other azure resources to the subnet1 that we use for the nodeType1.

From: Kay Singh [mailto:notifications@github.com]
Sent: Tuesday, December 1, 2015 2:46 PM
To: Azure/azure-quickstart-templates <azure-quickstart-templates@noreply.github.com>
Cc: Chacko Daniel <chackdan@microsoft.com>
Subject: Re: [azure-quickstart-templates] WAD 5 Node (#863)


In service-fabric-secure-cluster-5-node-1-nodetype-wad/azuredeploy.json<https://github.com/Azure/azure-quickstart-templates/pull/863#discussion_r46351665>:

> +      "location": "[variables('computeLocation')]",

> +      "properties": {

> +        "addressSpace": {

> +          "addressPrefixes": [

> +            "[variables('addressPrefix')]"

> +          ]

> +        },

> +        "subnets": [

> +          {

> +            "name": "[variables('subnet1Name')]",

> +            "properties": {

> +              "addressPrefix": "[variables('subnet1Prefix')]"

> +            }

> +          },

> +          {

> +            "name": "[variables('subnet2Name')]",

This subnet isn't being used anywhere, should be removed

—
Reply to this email directly or view it on GitHub<https://github.com/Azure/azure-quickstart-templates/pull/863/files#r46351665>.